keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resolvers/ApplicationLoadBalancerDefaultsResolver.kt

```diff
@@ -4,12 +4,12 @@ import com.netflix.spinnaker.keel.api.Resource
 import com.netflix.spinnaker.keel.api.ec2.ApplicationLoadBalancerSpec
 import com.netflix.spinnaker.keel.api.plugins.Resolver
 import com.netflix.spinnaker.keel.clouddriver.model.ApplicationLoadBalancerModel
-import com.netflix.spinnaker.keel.ec2.SPINNAKER_EC2_API_V1
+import com.netflix.spinnaker.keel.ec2.APPLICATION_LOAD_BALANCER_V1
 import org.springframework.stereotype.Component
 
 @Component
 class ApplicationLoadBalancerDefaultsResolver : Resolver<ApplicationLoadBalancerSpec> {
-  override val supportedKind = SPINNAKER_EC2_API_V1.qualify("application-load-balancer")
+  override val supportedKind = APPLICATION_LOAD_BALANCER_V1
```

Comment:
`ResourceHandler` implementations can now reference one of the constants to indicate what kind they support